contents/handbook/engineering/feature-ownership.md

```diff
@@ -20,12 +20,12 @@ You can also view the list [directly in GitHub](https://github.com/PostHog/posth
 |---|---|---|
 | Actions | [Team Product Analytics][Team Product Analytics]  | <span class="lemon-tag gh-tag">feature/actions</span> |
 | Annotations | [Team Product Analytics][Team Product Analytics]  | <span class="lemon-tag gh-tag">feature/annotations</span> |
-| API Structure | Security + core updates owned by [Team Pipeline][Team Pipeline]. Features owned by the relevant small team | <span class="lemon-tag gh-tag">feature/api-structure</span> |
-| Async migrations | [Team Pipeline][Team Pipeline]  | <span class="lemon-tag gh-tag">feature/async-migrations</span> |
+| API Structure | Security + core updates owned by [Team CDO][Team CDP]. Features owned by the relevant small team | <span class="lemon-tag gh-tag">feature/api-structure</span> |
+| Async migrations | [Team CDP][Team CDP]  | <span class="lemon-tag gh-tag">feature/async-migrations</span> |
 | Batch exports | [Team Batch Exports](/teams/batch-exports)  | <span class="lemon-tag gh-tag">feature/batch-exports</span> |
 | BI | [Team Data Warehouse](/teams/data-warehouse)  |  <span class="lemon-tag gh-tag">feature/dashboards</span> |
 | Billing | [Team Growth][Team Growth]  |  <span class="lemon-tag gh-tag">feature/billing</span> |
-| Client libraries | Core updates owned by [Team Pipeline][Team Pipeline]. Features owned by the relevant small team  | <span class="lemon-tag gh-tag">feature/pipeline</span> |
+| Client libraries | Core updates owned by [Team CDP][Team CDP]. Features owned by the relevant Small Team, or try #feature-client-libraries  | <span class="lemon-tag gh-tag">feature/pipeline</span> |
```

Comment:
```suggestion
| Client libraries | Core updates owned by [Team CDP][Team CDP]. Features owned by the relevant Small Team, or try #feature-client-libraries. There is an engineer assigned on a rotating schedule. Check [the (private) pager duty schedule](https://posthog.pagerduty.com/schedules#P7B7NTR)  | <span class="lemon-tag gh-tag">feature/pipeline</span> |
```

i would guess Team CDP don't want to own SDKs but interested to see what they say